docs/learning/18-java-static-analysis-stack.md

```
@@ -0,0 +1,176 @@
+---
+
+## 4. Gradle 설정 시행착오
+
+Error Prone Gradle 플러그인(`net.ltgt.errorprone`) 설정에서 3번 실패했다:
+
+1. `options.errorprone { option(...) }` → import 누락으로 "Unresolved reference"
+2. `options.errorprone.errorproneArgs.addAll(...)` → 잘못된 API
+3. `options.compilerArgs.addAll(...)` → Error Prone 플러그인과 호환 안 됨
+
+**올바른 방법:**
+```kotlin
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)